labs/accessing-optimizely-event-data/img/demo.yml

```diff
@@ -0,0 +1,145 @@
+# The configurations that used for the recording, feel free to edit them
```

Comment:
I assume this file will make more sense if I try to use oevents?

Reply:
No, this file was used to generate `demo.gif`; it was generated by a tool called terminalizer.  I changed the name of the config file and added some explanatory comments to the top to (hopefully) prevent confusion about this.